GSA-TTS/FAC
# Sk/3687 missing accounting policies

**Closes**: https://github.com/GSA-TTS/FAC/issues/3687, https://github.com/GSA-TTS/FAC/issues/3866

This PR handles blank rate_content, policies_content and leading '=' in policies_content.

**Testing**:
1. Test with a failed audit: (dbkey = 86892, audit_year = 2021) if access to census data in Preview is available.

- Run 'docker compose run --rm web python manage.py historic_data_migrator --years 'my_year' --dbkeys 'my_dbkey''
- Verify that accounting_policies in dissemination_note is set to 'GSA_MIGRATION'.
- Verify that note content in the migration inspection record is set to 'GSA_MIGRATION' in dissemination_migrationinspectionrecord table.

2. Test by changing note content in an audit:

- For an audit, in census eleccnotes table, where TYPE_ID=1, change CONTENT to ''.
- Run 'docker compose run --rm web python manage.py historic_data_migrator --years 'my_year' --dbkeys 'my_dbkey''
- Verify that accounting_policies in dissemination_note is set to 'GSA_MIGRATION'.
- Verify that note content in the migration inspection record is set to 'GSA_MIGRATION' in dissemination_migrationinspectionrecord table.

3. Test by changing note content in an audit:

- For an audit, in census eleccnotes table, where TYPE_ID=2, change CONTENT to ''.
- Run 'docker compose run --rm web python manage.py historic_data_migrator --years 'my_year' --dbkeys 'my_dbkey''
- Verify that rate_explained in dissemination_note is set to 'GSA_MIGRATION'.
- Verify that note content in the migration inspection record is set to 'GSA_MIGRATION' in dissemination_migrationinspectionrecord table.

4. Test by changing note content in an audit:

- For an audit, in census eleccnotes table, where TYPE_ID=1, add leading characters === to an existing CONTENT.
- Run 'docker compose run --rm web python manage.py historic_data_migrator --years 'my_year' --dbkeys 'my_dbkey''
- Verify that accounting_policies in dissemination_note does not have leading characters ===.
- Verify that transformation to note content is recorded in the migration inspection record in dissemination_migrationinspectionrecord table.

## PR checklist: submitters


## PR checklist: reviewers


The larger the PR, the stricter we should be about these points.
